orca-queue-sql/src/main/kotlin/com/netflix/spinnaker/config/SqlPendingExecutionConfiguration.kt

```diff
@@ -13,7 +14,7 @@ import org.springframework.context.annotation.Configuration
 import java.time.Clock
 
 @Configuration
-@EnableConfigurationProperties(SqlProperties::class)
+@EnableConfigurationProperties(com.netflix.spinnaker.kork.sql.config.SqlProperties::class)
```

Comment:
It's imported so shouldn't need the package name here?

Reply:
indeed, will remove